### PR TITLE
i make new profile  mohit.json to maahi.json

### PR DESCRIPTION
**Pull request by singhmohit14072002**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2304"></a>

### CODE REVIEW

**Review by srkds — CHANGES_REQUESTED** [review]
Hi @singhmohit14072002 

Thanks for contributing 🎉

Some changes and you're good to go.
- [ ] Format the file, proper indentation
- [ ] Rename the file to your GitHub username for uniqueness

Refer 👉[add your profile guide](https://github.com/EddieHubCommunity/LinkFree#-to-add-your-profile)

File: data/maahi.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://www.linkedin.com/in/mohit-singh-7964b1203/",
+        "icon": "github"
+    },
+    {
+        "name": "Twitter",
+        "url": "https://twitter.com/MohitDevops",
+        "icon": "twitter"
+    }
+    ]
+    }
```
Comment (eddiejaoude) [inline data/maahi.json:17]: The formatting is not quite right here, please fix and let's get this merged 👍

**Comment by eddiejaoude** [comment]
Json is valid, so I will merge

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
File name does not match GitHub username

**Comment by eddiejaoude** [comment]
I will close this PR as the file name does not match the username

**Pull request closed without merging**